src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -208,7 +208,7 @@ private function initializeSubscribers(): void
                 $listener = $this->container->get($listener);
             }
             // throw new \InvalidArgumentException(sprintf('Using Doctrine subscriber "%s" is not allowed, declare it as a listener instead.', \is_object($listener) ? $listener::class : $listener));
```

Comment:
```suggestion
            // throw new \InvalidArgumentException(sprintf('Using Doctrine subscriber "%s" is not allowed, declare it as a listener instead, using e.g. the attribute provided by the bundle.', \is_object($listener) ? $listener::class : $listener));
```
?

Reply:
Updated, this will make a conflict during upmerge.